src/test/ccl/ut_cclut_mock.inc

```diff
@@ -117,7 +117,7 @@ subroutine test_cclutAddMockImplementation_missing_originalName(null)
     call cclutAssertVcOperator(CURREF, "test_cclutAddMockImplementation_missing_originalName 001",
         trim(errorMessage, 3), "regexplike",
         concat("%CCL-E-392-PRG_[0-9]+_[0-9]+\([^)]+\)[0-9]+:[0-9]+\{CCLEXCEPTION\(\)\}Exception\(100\):",
-            " cclutAddMockImplementation\(\) - originalName is a required field."))
+            " cclutAddMockImplementation\(\) - originalName is a required field.")) ;^
```

Comment:
I see a couple of these ";^" in the file.  Is this a quirk in your editor?  Or do they serve some purpose?

Reply:
Yes. I can remove them.